livestream-compose-sample/src/main/java/io/getstream/livestream/compose/YoutubePlayer.kt

```diff
@@ -0,0 +1,32 @@
+package io.getstream.livestream.compose
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.YouTubePlayer
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.listeners.AbstractYouTubePlayerListener
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.options.IFramePlayerOptions
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView
+
+@Composable
+fun YTPlayer() {
+    AndroidView(modifier = Modifier.fillMaxSize(), factory = { ctx ->
```

Comment:
Modifiers shouldn't have default values! I know it's annoying to use them, but this way we avoid having our users misuse our components because they didn't realise they were using a default modifier.